granary/flickr.py

```diff
@@ -0,0 +1,371 @@
+from apiclient.errors import HttpError
```

Comment:
very low priority: maybe match the other files' import ordering? builtins, then deps, then this package. elsewhere too.